src/app/app.html

```diff
@@ -1,14 +1,17 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title>
+          PVTrAck
```

Comment:
[Elsewhere][1], we've done things to try to make this pronounced "PeeVeeTrack"

[1]:  https://github.com/umts/pvta-multiplatform/blob/master/src/index.html#L36